google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1522,14 +1526,14 @@ Blob create(
    * BlobInfo blobInfo = BlobInfo.newBuilder(blobId)
    *     .setContentType("text/plain")
    *     .build();
-   * Blob blob = storage.create(blobInfo, content, BlobWriteOption.encryptionKey(encryptionKey));
+   * Blob blob = storage.create(blobInfo, content, BlobTargetOption.encryptionKey(encryptionKey));
    * }</pre>
    *
    * @return a [@code Blob} with complete information
    * @throws StorageException upon failure
    */
-  @Deprecated
-  Blob create(BlobInfo blobInfo, InputStream content, BlobWriteOption... options);
+  Blob create(BlobInfo blobInfo, InputStream content, BlobTargetOption... options)
```

Comment:
This is also unrelated changes... This should be discussed with storage client maintainer because you are changing in end-user exposed client. 

@ajaaym Please let us know whom(at least for Storage) to discuss with for changing method signature in the surface client?